### PR TITLE
Update README with unoptimized version info

### PR DESCRIPTION
**Pull request by rebeccajfriedman**
Updates the README with an example on how to mod the JS Buy SDK queries.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +229,38 @@ client.checkout.removeDiscount(checkoutId).then(checkout => {
 });
 ```
 
+## Expanding the SDK
+
+Not all fields that are available on the [Storefront API](https://help.shopify.com/en/api/custom-storefronts/storefront-api/reference) are exposed through the SDK. If you use the unoptimized version of the SDK, you can easily build your own queries. In order to do this, use the UMD Unoptimized build.
+
+### Initializing the Client
+```javascript
+import Client from 'shopify-buy/umd';
```
Comment (rebeccajfriedman) [inline README.md]: How can I find out what this should be?

File: README.md [hunk 2]
```diff
@@ -73,6 +83,10 @@ import Client from 'shopify-buy/index.amd';
 ```javascript
 import Client from 'shopify-buy/index.umd';
 ```
+**UMD Unoptimized:**
```
Comment (minasmart) [inline README.md]: You should maybe include a definition of what unoptimized means. The term is very inside baseball as it is.

Comment (minasmart) [reply README.md]: At least in terms of the cases where this version is useful.

File: README.md [hunk 3]
```diff
@@ -215,6 +229,38 @@ client.checkout.removeDiscount(checkoutId).then(checkout => {
 });
 ```
 
+## Expanding the SDK
+
+Not all fields that are available on the [Storefront API](https://help.shopify.com/en/api/custom-storefronts/storefront-api/reference) are exposed through the SDK. If you use the unoptimized version of the SDK, you can easily build your own queries. In order to do this, use the UMD Unoptimized build.
+
+### Initializing the Client
```
Comment (minasmart) [inline README.md]: This should specify which client. By default, we want to recommend the optimized client, but for special cases, use the thicker one.

Comment (minasmart) [reply README.md]: (I know it's under the expanding the sdk section. But for the sake of people skimming or looking at just the headings it'd be useful)

**Pull request merged**